.github/workflows/python-package.yml

```diff
@@ -15,25 +15,25 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [2.7]
+        python-version: [3.6, 3.7, 3.8]
 
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - uses: actions/cache@v2
-      with:
-        path: ~/.cache/pip
-        key: ${{ runner.os }}-pip-${{ hashFiles('setup.py') }}
-        restore-keys: |
-          ${{ runner.os }}-pip-
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install tox
-    - name: Test with tox
-      # Run tox using the version of Python in `PATH`
+        pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
       run: |
-        tox -e py
+        pytest
```

Comment:
Is there a reason to get rid of `tox`? I find `tox` helpful because it makes it easier to run the same tests locally (including across multiple python versions and including the `flake8` linting) that are being run by CI.

Reply:
As mentioned before, I have switched to the default github action for testing python packages, We can include a tox file, if want to use tox for testing.

Reply:
As I mentioned, I think `tox` really helps with being able to run the tests locally without having to push it to GitHub first. Given that the work has already been done to enable `tox` in this project, I'd prefer to not rip it out.

Reply:
To be clear: I'm good with the switch to using `pytest`. Just make that switch inside the `tox.ini` file instead of in the GitHub Actions workflow, so that `tox` can be used to run `pytest`.